test/task.h

```diff
@@ -229,4 +231,61 @@ UNUSED static int can_ipv6(void) {
   "Cygwin runtime hangs on listen+connect in same process."
 #endif
 
+#if !defined(MAP_ANONYMOUS)
+#if defined(MAP_ANON)
+#define MAP_ANONYMOUS MAP_ANON
+#else
+#define MAP_ANONYMOUS 0 // MAP_FILE
```

Comment:
No C++/C99 comments.  That said, what UNIX platform does not have MAP_ANONYMOUS defined?  That said...

Reply:
This isn't exactly a C++/C99 comment (the parser warning is misleading), its a preprocessor comment. I didn't realize their behavior wasn't fully defined until the C99 standard however. Apparently I'll have to fix it anyways...

> That said...

Yes, getting this PR to pass the CI battery has been such wonderful fun...

(actually, `MAP_ANONYMOUS` is a very new flag – added to BSD-derived systems, including Darwin, in the last 10 years, from looking at man-page archeology – it was originally spelled `MAP_ANON`)